api/src/main/java/org/apache/iceberg/ContentFile.java

```diff
@@ -57,6 +57,9 @@
   /** Returns the file size in bytes. */
   long fileSizeInBytes();
 
+  /** Returns the file modified time in milliseconds. */
+  long fileModifiedTime();
```

Comment:
I'm curious whether `lastModifiedTimeMillis()` might be a slightly better name? Also we might want to consider to have this return `0L` by default, so that we don't introduce an API-breaking change

Reply:
`lastModifiedTimeMillis` sounds good to me. Let me rename relevant variables and methods. Thanks for your suggestion.